.gitignore

```diff
@@ -8,6 +8,7 @@ db/*.sqlite3
 /db/*.sqlite3-journal
 /log/*
 /tmp/*
+/bin/*
```

Comment:
we do wanna keep the `/bin/` directory checked into version control -- these are some key rails-related binaries. are they doing something weird on your machine?

Reply:
woops, i'll put em back.